Staged/Classes/STGNavigationControllerVerifier.h

```diff
@@ -0,0 +1,18 @@
+//
+//  STGNavigationControllerVerifier.h
+//  Staged
+//
+//  Created by Gustavo Barbosa on 7/31/16.
+//
+//
+
+#import <UIKit/UIKit.h>
+
+@interface STGNavigationControllerVerifier: NSObject
+
+@property (nonatomic, copy) NSArray<__kindof UIViewController *> * _Nonnull viewControllers;
```

Comment:
adding `nonnull` to the property instead? (`@property (nonatomic, copy, nonnull) ...`)

The same applies for `topViewController`